.github/workflows/lint-and-test.yml

```diff
@@ -3,8 +3,105 @@ name: lint-and-test
 on: [push, workflow_dispatch]
 
 jobs:
-  lint-and-test:
-    runs-on: ubuntu-22.04
+  test-current:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - 'ubuntu-22.04'
+          - 'ubuntu-24.04'
+
+    env:
+      elixir_version: '1.16'
+      otp_version: '25.3'
+
+    runs-on: ${{ matrix.os }}
+
+    services:
+      postgres:
+        image: postgres:10
+        env:
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: $otp_version
+          elixir-version: $elixir_version
```

Comment:
Typos:
```suggestion
          otp-version: ${{ env.otp_version }}
          elixir-version: ${{ env.elixir_version }}
```

Reply:
It appears the syntax I was using is allowed in steps, but not here. :-(  Thanks!